src/styles/blocks/_bike-card.scss

```diff
@@ -0,0 +1,34 @@
+.bike-card {
+  width: 280px;
+
+  @include onTablet {
+    @include cardSize(392px);
+  }
+
+  @include onDesktop {
+    @include cardSize(33%);
+  }
```

Comment:
You can add width to variable, or use grids instead of hardcoding element width.